apis/userapi/userapi.yaml

```diff
@@ -1799,6 +1891,28 @@ definitions:
             type: string
             description: userId
 
+  MigrateUserReq:
+    description: request body for migrating user
+    type: object
+    properties:
+      request:
+        type: object
+        properties:
+          userId:
+            type: string
+            description: userId of migrating user
+          action:
+            type: string
+            description: accept/reject only
+          feedId:
+            type: string
+            description: notificationId //optional
+          channel:
+            type: string
+            description: to which channel user wants to migrate
+          userExtId:
+            type: string
+            description: valid value of userExtId should be provided in order to get migrate.
```

Comment:
User's external identifier. This must match with the value provided at the time of bulk user upload.

Reply:
done